setup.py

```diff
@@ -472,12 +472,12 @@ def readme():
 # in the correct place unless we use 'package_dir' and this trick.
 # 1. Here we list files as they should appear in packages for end-users
 # 2. In 'package_dir' we defer the package name to the local file path
-packages += map(lambda x: f"sisl_toolbox.{x}", find_packages("toolbox"))
+packages += map(lambda x: f"toolbox.{x}", find_packages("toolbox"))
```

Comment:
Why are you changing this?

Reply:
It couldn't find toolbox folder i think probably u changed the name from `sisl_toolbox` ----> `toolbox`...

Reply:
Please remove it from this pr and make a new issue. I don't have problems with this, so I think you either do something wrong or misinterpret what this does?

Reply:
humm i don't know may be but it doesn't find the toolbox then to install it unless there is another way to install sisl (so when u just pip install ... it should install the toolbox ... )

Reply:
And I am pretty sure it does. I have checked with conda. But I'll check some more, in any case it does not belong in this pr.

Reply:
Also, it works seemleasly for me

Reply:
okay found my mistake u r correct things are fine probably the doc have some type in doc (http://zerothi.github.io/sisl/docs/latest/toolbox/index.html) is:
`import sisl_toolbox.siesta.atom`
while it should be:
`import toolbox.siesta.atom`

Reply:
No, no typo it works with that module name

Reply:
This is the way python ensures renaming folders gets translated to the correct module name.